zephyr/subsys/mcuboot_app/CMakeLists.txt

```diff
@@ -0,0 +1,234 @@
+set(BOOT_DIR ${ZEPHYR_MCUBOOT_MODULE_DIR}/boot)
+# Path to top-level repository root directory.
+set(MCUBOOT_DIR ${ZEPHYR_MCUBOOT_MODULE_DIR})
+# Path to tinycrypt library source subdirectory of MCUBOOT_DIR.
+set(TINYCRYPT_DIR "${ZEPHYR_MCUBOOT_MODULE_DIR}/ext/tinycrypt/lib")
+assert_exists(TINYCRYPT_DIR)
+set(TINYCRYPT_SHA512_DIR "${ZEPHYR_MCUBOOT_MODULE_DIR}/ext/tinycrypt-sha512/lib")
+assert_exists(TINYCRYPT_SHA512_DIR)
+# Path to crypto-fiat
+set(FIAT_DIR "${ZEPHYR_MCUBOOT_MODULE_DIR}/ext/fiat")
+assert_exists(FIAT_DIR)
+# Path to mbed-tls' asn1 parser library.
+set(MBEDTLS_ASN1_DIR "${ZEPHYR_MCUBOOT_MODULE_DIR}/ext/mbedtls-asn1")
+assert_exists(MBEDTLS_ASN1_DIR)
+set(NRF_DIR "${ZEPHYR_MCUBOOT_MODULE_DIR}/ext/nrf")
+
+if(CONFIG_BOOT_USE_NRF_CC310_BL)
+set(NRFXLIB_DIR ${ZEPHYR_NRFXLIB_MODULE_DIR})
+if(NOT EXISTS ${NRFXLIB_DIR})
+  message(FATAL_ERROR "
+    ------------------------------------------------------------------------
+    No such file or directory: ${NRFXLIB_DIR}
+    The current configuration enables nRF CC310 crypto accelerator hardware
+    with the `CONFIG_BOOT_USE_NRF_CC310_BL` option. Please follow
+    `ext/nrf/README.md` guide to fix your setup or use tinycrypt instead of
+    the HW accelerator.
+    To use the tinycrypt set `CONFIG_BOOT_ECDSA_TINYCRYPT` to y.
+    ------------------------------------------------------------------------")
+endif()
+# Don't include this if we are using west
+ add_subdirectory(${NRFXLIB_DIR} ${PROJECT_BINARY_DIR}/nrfxlib)
+endif()
+
+zephyr_library_include_directories(
+  ${ZEPHYR_MCUBOOT_MODULE_DIR}/zephyr/include
+  targets
+  )
+if(EXISTS targets/${BOARD}.h)
+  zephyr_library_compile_definitions(MCUBOOT_TARGET_CONFIG="${BOARD}.h")
+endif()
+
+# Zephyr port-specific sources.
+zephyr_library_sources(
+  ${BOOT_DIR}/zephyr/main.c
+  ${BOOT_DIR}/zephyr/flash_map_extended.c
+  ${BOOT_DIR}/zephyr/os.c
+  )
+
+if(DEFINED CONFIG_ENABLE_MGMT_PERUSER)
+  zephyr_library_sources(
+    ${BOOT_DIR}/zephyr/boot_serial_extensions.c
+    )
+endif()
+
+if(NOT DEFINED CONFIG_FLASH_PAGE_LAYOUT)
+  zephyr_library_sources(
+    ${BOOT_DIR}/zephyr/flash_map_legacy.c
+    )
+endif()
+
+# Generic bootutil sources and includes.
+zephyr_library_include_directories(${BOOT_DIR}/bootutil/include)
+zephyr_library_sources(
+  ${BOOT_DIR}/bootutil/src/image_validate.c
+  ${BOOT_DIR}/bootutil/src/tlv.c
+  ${BOOT_DIR}/bootutil/src/encrypted.c
+  ${BOOT_DIR}/bootutil/src/image_rsa.c
+  ${BOOT_DIR}/bootutil/src/image_ec256.c
+  ${BOOT_DIR}/bootutil/src/image_ed25519.c
+  ${BOOT_DIR}/bootutil/src/bootutil_misc.c
+  ${BOOT_DIR}/bootutil/src/fault_injection_hardening.c
+  )
+
+if(CONFIG_BOOT_FIH_PROFILE_HIGH)
+zephyr_library_sources(
+  ${BOOT_DIR}/bootutil/src/fault_injection_hardening_delay_rng_mbedtls.c
+  )
+endif()
+
+if(CONFIG_SINGLE_APPLICATION_SLOT)
+zephyr_library_sources(
+  ${BOOT_DIR}/zephyr/single_loader.c
+  )
+zephyr_library_include_directories(${BOOT_DIR}/bootutil/src)
```

Comment:
Missing indentation.

Reply:
This repository is not the Zephyr repository nor under the control of the Zephyr Project. Has the MCUboot project defined a source file formatting standard or policy?

Reply:
No, but it does not mean that indentations have to be chaotic.
This is moved code anyway, so it is how it is. No change required.

Reply:
Agreed.  It should be cleaned up in a different PR.